worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +54,45 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	int Likeness = 0;
+	int IncorrectGuesses = 0;
```

Comment:
Inconsistent naming -- some of your variable names start with uppercase letters, some with lowercase.